components/front-page/LearnSection.tsx

```diff
@@ -14,19 +16,35 @@ const LearnCard = ({
   description,
   className,
   link,
+  img,
 }: LearnCardProps) => {
   return (
     <motion.div
-      whileHover={{ scale: 1.03 }}
+      whileHover={{
+        scale: 1.03,
+        boxShadow: "0px 10px 20px 5px rgba(0,0,0,0.25)",
+      }}
       whileTap={{ scale: 1 }}
-      className={`${className} min-w-[220px] w-full h-[126px] rounded-[10px] p-[15px]`}
+      animate={{}}
```

Comment:
No, it's required because the classes from the individual card props need to be passed through, but I'll change the prop name to avoid confusion though.